packages/iTwinUI-react/src/core/ComboBox/ComboBox.test.tsx

```diff
@@ -480,3 +480,40 @@ it('should use the latest onChange prop', async () => {
   expect(mockOnChange2).toHaveBeenNthCalledWith(1, 2);
   expect(mockOnChange1).toHaveBeenCalledTimes(1);
 });
+
+it('should call onExpand and onCollapse when dropdown is opened and closed', () => {
+  const onExpand = jest.fn();
+  const onCollapse = jest.fn();
+  const { container } = renderComponent({
+    onExpand,
+    onCollapse,
+  });
+
+  const icon = container.querySelector('.iui-end-icon svg') as HTMLElement;
+  fireEvent.click(icon);
+  const list = document.querySelector('.iui-menu') as HTMLUListElement;
+  expect(list).toBeVisible();
+  expect(onExpand).toHaveBeenCalled();
+
+  fireEvent.click(icon);
+  expect(list).not.toBeVisible();
+  expect(onCollapse).toHaveBeenCalled();
+});
+
+it('should render custom empty message element', async () => {
```

Comment:
nit: it's not very obvious that this is testing the overloading logic, maybe rename

```suggestion
it('should accept ReactNode in emptyStateMessage', async () => {
```